src/commands/env/compute/collaborator/add.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import herokuColor from '@heroku-cli/color';
+import * as Heroku from '@heroku-cli/schema';
+import { Errors, Flags } from '@oclif/core';
+import { cli } from 'cli-ux';
+import { Messages } from '@salesforce/core';
+import { FunctionsFlagBuilder } from '../../../../lib/flags';
+import Command from '../../../../lib/base';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-functions', 'env.compute.collaborator.add');
+
+export default class ComputeCollaboratorAdd extends Command {
+  static summary = messages.getMessage('summary');
+
+  static examples = messages.getMessages('examples');
+
+  static flags = {
+    'heroku-user': FunctionsFlagBuilder.environment({
+      char: 'h',
+      description: messages.getMessage('flags.heroku-user.summary'),
+      required: true,
+    }),
+  };
+
+  async run() {
+    const { flags } = await this.parse(ComputeCollaboratorAdd);
+    const herokuUser = flags['heroku-user'];
+
+    if (!herokuUser) {
+      throw new Errors.CLIError(
+        `Missing required flag:
+        -c, --heroku-user heroku-email  ${herokuColor.dim('Heroku user email address.')}
```

Comment:
Related to the above comment, it's using `-c` here, is that what it should be instead of `-h`?